src/store/calendar.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Maps a dav collection to our calendar object model
```

Comment:
*our* model is the one used in Calendar, right? unsure if we want to push that into calendar-js. @st3iny what do you think?

Reply:
we will leave it as-is